frontend/app/routes/public/renew/$id/ita/exit-application.tsx

```diff
@@ -69,11 +69,11 @@ export default function RenewItaExitApplication() {
           params={params}
           disabled={isSubmitting}
           startIcon={faChevronLeft}
-          data-gc-analytics-customclick="ESDC-EDSC:CDCP Renew Application Form-Adult:Back - Exiting the application click"
+          data-gc-analytics-customclick="ESDC-EDSC:CDCP Renew Application Form-ITA:Back - Exit application click"
```

Comment:
(Same comment for the rest of the attribute values in this route)
```suggestion
          data-gc-analytics-customclick="ESDC-EDSC:CDCP Renew Application Form-ITA:Back - Exiting the application click"
```